lib/generator._js

```diff
@@ -458,18 +454,17 @@ ScriptGenerator.prototype.generateDotNetDeploymentScript = function (templateFil
 ScriptGenerator.prototype.generateAspNetCoreScript = function (templateFileName, options, _) {
   argNotNull(templateFileName, 'templateFileName');
 
-  var fixedAspNetCoreProject = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.aspNetCoreProject) : fixPathSeperatorToUnix(options.aspNetCoreProject);
-  var fixedRestore = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.restore) : fixPathSeperatorToUnix(options.restore);
-  var fixedSolutionPath = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.solutionPath) : fixPathSeperatorToUnix(options.solutionPath);
+  var fixedDotnetpublishArguments = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.DotnetpublishArguments) : fixPathSeperatorToUnix(options.DotnetpublishArguments);
+  var fixedRestoreArguments = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.RestoreArguments) : fixPathSeperatorToUnix(options.RestoreArguments);
```

Comment:
when I was working on msbuild15 template, the fix for the linux path was not yet out...so I didn't correctly rename them. But since every property in `options` object is a path, we can probably write a for loop to reformat them in place instead of this:
```
-]).replace(/{DotnetpublishArguments}/g, options.DotnetpublishArguments)
 -    .replace(/{RestoreArguments}/g, options.RestoreArguments)
 +]).replace(/{DotnetpublishArguments}/g, fixedDotnetpublishArguments)
 +    .replace(/{RestoreArguments}/g, fixedRestoreArguments)
```
but again, the changes are now on hold due to msbuild15 discovery, so maybe later?